_config_production.yml

```diff
@@ -1,2 +1,4 @@
 domain: www.vets.gov
 destination:  ./_site_production
+
+exclude: [dummy-placeholder.html]
```

Comment:
We need to do it backwards for now...make the file default hidden...explicitly include it in `_config.yml` and then remove that `include` in `_config.production.yml` (which only works cause we don't use `include`.